app/src/test/java/com/futurice/freesound/network/api/DefaultFreeSoundSearchServiceTest.java

```diff
@@ -62,11 +61,10 @@ public void setUp() {
     public void search_emitsResults_whenApiSuccessful() {
         new Arrangement().withApiResultResult(RESULT);
 
-        TestSubscriber<SoundSearchResult> ts =
-                defaultFreeSoundSearchService.search(QUERY).test();
-
-        ts.assertNoErrors()
-          .assertValue(RESULT);
+        defaultFreeSoundSearchService.search(QUERY)
+                                     .test()
+                                     .assertNoErrors()
```

Comment:
I think I've historical added this assertion because it gave better debug information when the test failed. At least on some 1.x versions, the failed assertion still left you in the dark without it.